pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
 ---
-title: Build an Image
+title: Build a Container
 sidebar: user_docs
 permalink: docs-build
 folder: docs
+toc: false
 ---
 
-Build is the process where we install an operating system and then configure it appropriately for a specified need. To do this we use a [Singularity recipe](/docs-recipes) file (a text file called `Singularity`) which is a recipe of how to specifically build the container. Here we will overview the usage, best practices, and quick examples.
+The `build` command lets you obtain a container. 
 
+`build` is the "Swiss army knife" of container creation.  You can use it to download and assemble pre-built containers from external resources like [Singularity Hub](https://singularity-hub.org/) and [Docker Hub](https://hub.docker.com/).  You can use it to convert containers between the various formats supported by Singularity.  And you can use it in conjunction with a [Singularity Recipe](/docs-recipes) file to create a container from scratch and customized it to fit your needs. 
 
-**Getting Started**
-For a detailed **walkhrough of getting started** with build, we will point you to the <a href="/quickstart">quickstart</a>. 
+{% include toc.html %}
 
-**Build Environment**
-If you want to customize the cache, specify Docker credentials, or any custom tweaks to your build environment, see <a href="/build-environment">build environment</a>. 
+## Overview
+The `build` command accepts a target as input and produces a container as output.  
 
-**Modular Containers**
-If you want to look at how to get started to making interally **modular containers**, check out <a href="http://containers-ftw.org/apps/examples/tutorials/getting-started" target="_blank">this post</a>.
+The target defines the method that `build` uses to create the container. It can be one of the following:
 
+- URI beginning with **shub://** to build from Singularity Hub
+- URI beginning with **docker://** to build from Docker Hub
+- path to a **existing container** on your local machine
+- path to a **directory** to build from a sandbox
+- path to an **archive** in .tar or compressed .tar.gz format
+- path to a **[Recipe file](docs-recipes)**
 
-{% include toc.html %}
+In addition `build` can produce containers in three different formats.  Formats types can be specified by passing the following options to build.  
 
-## Bases
```

Comment:
Can we still have a header for this? Eg `## Container Formats` so the user could jump to it quickly?